src/native/managed/cdacreader/src/SOSDacImpl.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Marshalling;
+
+namespace Microsoft.Diagnostics.DataContractReader;
+
+[GeneratedComInterface]
+[Guid("4eca42d8-7e7b-4c8a-a116-7bfbf6929267")]
+internal partial interface ISOSDacInterface9
+{
+    void GetBreakingChangeVersion(out int version);
```

Comment:
```suggestion
    int GetBreakingChangeVersion();
```

I think `out` parameters in this case should be avoided. I would prefer we treat the `retval` in the most C# manner.